person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if @age >= 18 || @parent_permission
+    false
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ ] @GutemaG I saw your message concerning changing of the name, since its a project requirement we are not supposed to change the name, you could try to solve the error, and if not possible as a final resort diable linter on this particular area of code.